packages/react-popover/src/popoverContext.ts

```diff
@@ -1,7 +1,7 @@
 import { createContext, useContextSelector, ContextSelector, Context } from '@fluentui/react-context-selector';
 import { PopoverState } from './components/Popover/index';
 
-export const PopoverContext: Context<PopoverContextValue> = createContext<PopoverContextValue>({
+export const popoverDefaultContext: PopoverContextValue = {
```

Comment:
We are using this only in tests, correct? Can we modify reexports to avoid leaking this as public export?

![image](https://user-images.githubusercontent.com/14183168/119480519-0d1e7580-bd52-11eb-95b0-b85541483e64.png)

Reply:
sure